drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/pages/ActionInsertFactFieldsPageViewImpl.ui.xml

```diff
@@ -26,90 +27,99 @@
   <ui:with field="res" type="org.uberfire.ext.widgets.core.client.resources.WizardResources"/>
 
   <gwt:VerticalPanel ui:field="container" width="100%">
```

Comment:
Fair enough.. this PR was just to fix the Guided Decision Table Wizard - that was broken (style-wise) after the BS3/PF migration. We have many, many uses of GWT's "layout" widgets (VerticalPanel, HorizontalPanel, SimplePanel etc) throughout the workbench (and Uberfire)... replacing them all would be a sizeable undertaking!